GPBuild

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "1.0",
+  "rules": {
+    "root": "",
+    "flags": "-DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DBUILD_NUMBER=0 -DBUILD_CHANNEL=DEVELOPER -DEVENT__DISABLE_TESTS=ON -DDOCTEST_WITH_TESTS=OFF"
```

Comment:
-DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DBUILD_NUMBER=0 -DBUILD_CHANNEL=DEVELOPER has forward mode top level layers (apps), you can remove these and keep only doctest specific